project/Build.scala

```diff
@@ -115,7 +115,7 @@ object ApplicationBuild extends Build {
   //   libraryDependencies ++= provided(play.api, reactivemongo.driver)
   // )
 
-  lazy val common = project("common").settings(
+  lazy val common = project("common", Seq(chess)).settings(
```

Comment:
aww, that's the only sad thing about this otherwise great PR. I liked the decoupled standalone `common` module. But whatever, let's move on!